cage.1.scd

```diff
@@ -6,7 +6,7 @@ cage - a Wayland kiosk compositor
 
 # SYNOPSIS
 
-*cage* [-dhmrsv] [--] _application_ [application argument ...]
+*cage* [-dhmrsv] [--] [_application_...]
```

Comment:
It makes me realize that `--` usage to split Cage arguments from the one passed to child process is not documented in the man page. I found it only in the [wiki](https://github.com/cage-kiosk/cage/wiki/Configuration). May be interesting to add a line here also.

Reply:
Right. It's unrelated to this PR though.